.circleci/config.yml

```diff
@@ -188,7 +194,7 @@ jobs:
           command: |
             python -m venv env
             . env/bin/activate
-            pip install dist/dwave-gate*.tar.gz
+            pip install dist/dwave_gate*.tar.gz
```

Comment:
For some reason sdist started naming using and underscore instead of dash. I vaguely familiar with these being interchangeable with Python builds, but not sure about the details there.

Reply:
Yeah, I ran into the same thing in another PR :shrug: